app/services/ingest_iati_activities.rb

```diff
@@ -279,6 +280,11 @@ def call
     end
   end
 
+  private def add_intended_beneficiaries(legacy_activity:, new_activity:)
+    # To be ingested once we have the information
+    new_activity.intended_beneficiaries = ["Replace me"]
```

Comment:
Will there be any side effects when editing an activity whose `intended_beneficiaries` has been set like this?

Do we need to set this value for the activities already ingested?

Reply:
For activities with a value of `recipient_country` on `geography` step we could use "Not Applicable". For activities with `recipient_region` I don't have a better idea than this placeholder. Open to suggestions, though